src/app/page.tsx

```diff
@@ -227,10 +247,10 @@ export default function Home() {
   // selected, and hardcoded data for the rest of the inputs
   const handleGenerateHDRImage = () => {
     invoke<string>("pipeline", {
-      radiancePath: settings.radiancePath,
-      hdrgenPath: settings.hdrgenPath,
-      outputPath: settings.outputPath,
-      tempPath: settings.tempPath,
+      radiancePath: fakeRadiancePath,
```

Comment:
It looks like some hardcoded arguments accidentally got added back in - I think we'll want to remove these before we merge to main